x-pack/plugin/esql/compute/src/main/generated-src/org/elasticsearch/compute/data/LongVectorFixedBuilder.java

```diff
@@ -25,6 +25,8 @@ final class LongVectorFixedBuilder implements LongVector.FixedBuilder {
      */
     private int nextIndex;
```

Comment:
Is it possible to drop this entirely? If, after your PR, it is still used, then we should do it in a followup.